webapi/_test_data/ontologies/beol-onto.ttl

```diff
@@ -1311,6 +1307,134 @@
                    rdfs:comment """Der Brief ist publiziert"""@de ,
                               """Letter is published"""@en .
 
+### ###########################################
+### beol:transcriptionOf
+
+    :transcriptionOf rdf:type owl:ObjectProperty ;
+
+           rdfs:subPropertyOf knora-base:hasLinkTo ;
+
+           rdfs:comment "Transkription von"@de ,
+                        "Transcription of"@en ;
+
+           rdfs:label  "Transkription von"@de ,
+                       "Transcription of"@en ;
+
+           knora-base:subjectClassConstraint :transcription ;
+
+           knora-base:objectClassConstraint :manuscriptEntry ;
+
+           salsah-gui:guiElement salsah-gui:Searchbox .
+
+
+### ###########################################
+### beol:transcriptionOfValue
+
+:transcriptionOfValue rdf:type owl:ObjectProperty ;
+
+                knora-base:objectClassConstraint knora-base:LinkValue ;
+
+                knora-base:subjectClassConstraint :transcription ;
+
+                rdfs:subPropertyOf knora-base:hasLinkToValue ;
+
+                rdfs:comment "Transkription von"@de ,
+                             "Transcription of"@en ;
+
+                rdfs:label  "Transkription von"@de ,
+                            "Transcription of"@en .
+
+### ###########################################
+### beol:belongsToRegion
+
+    :belongsToRegion rdf:type owl:ObjectProperty ;
+
+           rdfs:subPropertyOf knora-base:hasLinkTo ;
+
+           rdfs:comment "Transkription von"@de ,
```

Comment:
belongsToRegion has same comment and label as the property transcriptionOf.

Reply:
done in 
https://github.com/dhlab-basel/Knora/pull/890/commits/60740e5d27128b51d7dcb259abfe23ba8e6612f9